scripts/variantstore/wdl/extract/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM python:3.7
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:305.0.0
 
 # Copy the application's requirements.txt and run pip to install
 ADD requirements.txt /app/requirements.txt
 RUN pip install -r /app/requirements.txt
 
 # Add the application source code.
 ADD raw_array_cohort_extract.py /app
```

Comment:
can you remove this from Dockerfile, it's going away soon (and we don't use it).  You can leave the `raw_array_cohort_extract.py` file still, we'll get that with the arrays cleanup

Reply:
ah super, sure